src/lib/components/chat/Settings/Account.svelte

```
@@ -205,7 +205,7 @@
 						<button
 							class=" text-xs text-center text-gray-800 dark:text-gray-400 rounded-lg px-2 py-1"
 							on:click={async () => {
-								profileImageUrl = '/user.png';
+								profileImageUrl = `${WEBUI_URL}/user.png`;
```

ESLint

[error] 'WEBUI_URL' is not defined.

(no-undef)